<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoughDialog</class>
 <widget class="QDialog" name="HoughDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hough transform</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Method</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="method1Button">
        <property name="text">
         <string>Method #1</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="method2Button">
        <property name="text">
         <string>Method #2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="angleLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Angle step : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="angleBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="distanceLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Distance step : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="distanceBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>512.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HoughDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HoughDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method2Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>angleLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method2Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>angleBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method2Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>distanceLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>method2Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>distanceBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
